apps/website/screens/components/textarea/code/TextareaCodePage.tsx

```diff
@@ -277,6 +277,17 @@ const sections = [
             <td>Reference to the component.</td>
             <td>-</td>
           </tr>
+          <tr>
+            <td>ariaLabel</td>
+            <td>
+              <TableCode>string</TableCode>
+            </td>
+            <td>
+              A string of text that will be the accessible name for the textarea. Consider using this prop in case there
```

Comment:
I think this description can be improved: "Specifies a string to be used as the name for the textarea element when no <Code>label</Code> is provided."

What do you guys think? @Jialecl @Mil4n0r